tracer/src/Datadog.Trace/Telemetry/Metrics/MetricTags.cs

```diff
@@ -29,6 +29,7 @@ internal enum InitializationComponent
         [Description("component:traceattributes_pinvoke")] TraceAttributesPinvoke,
         [Description("component:managed")] Managed,
         [Description("component:calltarget_defs_pinvoke")] CallTargetDefsPinvoke,
+        [Description("component:serverless")] Serverless,
```

Comment:
I'm not sure about this change? 🤔 Is it being used? If not, it shouldn't be added, as it adds allocation to the metrics. If it _is_ you need to build locally to regenerate all the metrics

Reply:
It was removed by the PR https://github.com/DataDog/dd-trace-dotnet/pull/4535. I saw the pipeline failure related to Telemetry tests so just want to see if it's related.

Reply:
Updated the expected `DistributionShared.InitTime` by 1 in the `Telemetry_SendsMetrics` test since the Serverless component is now removed.